index.html

```diff
@@ -0,0 +1,198 @@
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-3">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li  class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-4">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+            </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-5">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-6">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <h2 class="center">About me</h2>
+      <p class="center">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="cont">
+        <button class="see center-button">Get My Resume</button>
+      </div>
+      <img class="about-background1" src="Images/Mask Group.svg" alt="">
+      <img class="about-background2" src="Images/Frame.svg" alt="">
```

Comment:
- kindly, those images are not part of the main content so If an image is not part of the content (aesthetic reasons) use CSS background-image. as the previous code reviewer suggested you [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. :+1:

Reply:
fixed